scripts/github-changelog.php

```diff
@@ -136,6 +138,10 @@ function create_draft_changelog( $title, $content, $tags ) {
         'tags' => implode( ',', $tags ),
     ];
 
+    if ( $channels ) {
```

Comment:
Since this is an array, should we check for array length instead?

Reply:
We could but empty array is falsy in `php`. So I think this is fine (I also tested that it doesn't add the new field if you don't send in the parameter).

Reply:
> empty array is falsy in php

Nice!